x-pack/plugins/apm/server/routes/alerts/rule_types/transaction_duration/register_transaction_duration_rule_type.ts

```diff
@@ -105,6 +103,16 @@ export function registerTransactionDurationRuleType({
     minimumLicenseRequired: 'basic',
     isExportable: true,
     executor: async ({ params: ruleParams, services, spaceId }) => {
+      const predefinedGroupby = [
+        SERVICE_NAME,
+        SERVICE_ENVIRONMENT,
+        TRANSACTION_TYPE,
+      ];
+
+      const allGroupbyFields = Array.from(
+        new Set([...predefinedGroupby, ...(ruleParams.groupBy ?? [])])
+      );
```

Comment:
nit: this can be simplified like:

```ts
      const allGroupbyFields = union(predefinedGroupBy, ruleParams.groupBy)
```

(also: there is a consistency with "group by". Sometimes it's written as "groupby" other times as "groupBy". I suggest we consistently use the latter.

Reply:
I will address this in a new PR.